build/build.yml

```diff
@@ -143,74 +137,8 @@ steps:
 
-# Clean target folder and create netcore 3.1 and netfx releases
-- script: dotnet publish CredentialProvider.Microsoft/CredentialProvider.Microsoft.csproj --no-build --framework netcoreapp3.1 --configuration $(BuildConfiguration) -p:${{ parameters.nuspecProperties }}
-  displayName: dotnet publish netcoreapp3.1
-
-- task: CopyFiles@2
-  displayName: Copy netcore 3.1 files into tarball
-  inputs:
-    sourceFolder: '$(Build.SourcesDirectory)\CredentialProvider.Microsoft\bin\$(BuildConfiguration)\netcoreapp3.1\publish\'
-    contents: '**\*'
-    targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\plugins\netcore\CredentialProvider.Microsoft\'
-    cleanTargetFolder: true
-
-- task: CopyFiles@2
-  displayName: Copy license files into tarball
-  inputs:
-    contents: |
-      LICENSE
-      CredentialProvider.Microsoft\EULA_Microsoft Visual Studio Team Services Credential Provider.docx
-      CredentialProvider.Microsoft\ThirdPartyNotices.txt
-    targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\'
-    flattenFolders: true
-
-- task: ArchiveFiles@2
-  displayName: Create netcore 3.1 tarball
-  inputs:
-    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
-    includeRootFolder: false
-    archiveType: 'tar'
-    tarCompression: 'gz'
-    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NetCore3.NuGet.CredentialProvider.tar.gz'
-    replaceExistingArchive: true
-
-# NOTE: Changing the name of the zip will break things. Please don't do it.
-- task: ArchiveFiles@2
-  displayName: Create netcore 3.1 zip
-  inputs:
-    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
-    includeRootFolder: false
-    archiveType: 'zip'
-    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NetCore3.NuGet.CredentialProvider.zip'
-    replaceExistingArchive: true
-
-- task: CopyFiles@2
-  displayName: Copy netfx files into tarball
-  inputs:
-    sourceFolder: '$(Build.SourcesDirectory)\CredentialProvider.Microsoft\bin\$(BuildConfiguration)\net461\'
-    contents: '**\*'
-    targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\plugins\netfx\CredentialProvider.Microsoft\'
-
-- task: ArchiveFiles@2
-  displayName: Create netfx tarball
-  inputs:
-    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
-    includeRootFolder: false
-    archiveType: 'tar'
-    tarCompression: 'gz'
-    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NuGet.CredentialProvider.tar.gz'
-    replaceExistingArchive: true
-
+# Clean target folder and create netfx releases
 # NOTE: Changing the name of the zip will break things. Please don't do it.
-- task: ArchiveFiles@2
-  displayName: Create zip
-  inputs:
-    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
-    includeRootFolder: false
-    archiveType: 'zip'
-    archiveFile: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\Microsoft.NuGet.CredentialProvider.zip'
```

Comment:
I think not producing a Microsoft.NuGet.CredentialProvider.zip/tar assets will break things - we might need to rename net 6 or net 8 to Microsoft.NuGet.CredentialProvider.zip/tar

Reply:
The NuGet plugin should rely on .NET Framework only correct? We're still publishing `Microsoft.NuGet.NetFx48.CredentialProvider.zip`